src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourceItem.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.Options
+{
+    public class PackageSourceItem
```

Comment:
I made PackageSourceItem so it is easier to add only sources that are checked in the`AddMappingDialog` to the config

Reply:
Can we rename this to be more specific? "Package" and "Item" are very overused in our repo already.